ros_buildfarm/argument.py

```diff
@@ -63,6 +63,13 @@ def add_argument_build_name(parser, build_file_type):
              build_file_type)
 
 
+def add_argument_env_vars(parser):
+    parser.add_argument(
+        '--env-vars',
+        nargs='*',
+        help="Environment variables as 'key=value' for Dockerfile ENV directives")
```

Comment:
This line exceeds the limit. I'm not sure how best to break up strings like this.

Reply:
The same file contains this kind of case (e.g. https://github.com/ros-infrastructure/ros_buildfarm/blob/728cfe8e3c40ecd4e0128030d1d66864816626c8/ros_buildfarm/argument.py#L167-L168). So just use the same style here.